aspnet/client-side/bower/samples/Starter/.bowerrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
the `bowerc` file? or the entire Starter sample? The Starter sample is what is used in the entire beginning of the article of how to add the resources etc. 

There is not a link to the sample code if that is what you mean